dpnp/dpnp_iface_linearalgebra.py

```diff
@@ -292,8 +291,12 @@ def vdot(*args, **kwargs):
 
     For full documentation refer to :obj:`numpy.vdot`.
 
+    Notes
+    -----
+    This function works the same as :obj:`dpnp.dot`
```

Comment:
```suggestion
    This function works the same as :obj:`dpnp.dot`.
```
`Notes` should be located after `See Also` according to the template.